src/utils/include/com/amazonaws/kinesis/video/utils/Include.h

```diff
@@ -995,8 +995,19 @@ BOOL checkBufferValues(PVOID, BYTE, SIZE_T);
  */
 PUBLIC_API STATUS generateTimestampStr(UINT64, PCHAR, PCHAR, UINT32, PUINT32);
 
+/**
+ * @PCHAR   - IN - timestamp format string
```

Comment:
Since this is a Public API can we provide sample usage here so it's clear that the user isn't expected to pass in the milliseconds part as part of the timestamp format string, because from the name of the function it is not clear that the user of this API is not expected to pass in the milliseconds.

Reply:
Fair. I fixed the documentation for this function and `generateTimestampStr`. Provided a sample and input expectations.